### PR TITLE
Update dependencies

### LINKED ISSUE #20: Update dependencies

### PR DESCRIPTION
**Pull request by brandon-welsch**
Close #20

See commits log for more details.

Tests passed:
```go
$ go test ./...
ok      github.com/Scalingo/go-handlers 0.004s
```

### CODE REVIEW

File: error_middleware.go [hunk 1]
```diff
@@ -29,7 +30,7 @@ var ErrorMiddleware MiddlewareFunc = MiddlewareFunc(func(handler HandlerFunc) Ha
 				}
 				log.WithError(err).Error("recover panic")
 				w.WriteHeader(500)
-				fmt.Fprintln(w, err)
+				_, _ = fmt.Fprintln(w, err)
```
Comment (EtienneM) [inline error_middleware.go]: I don't like these

Comment (brandon-welsch) [reply error_middleware.go]: Should I go back over the others with `json.NewEncoder...` ?

Comment (EtienneM) [reply error_middleware.go]: Yes please

Comment (brandon-welsch) [reply error_middleware.go]: Done

**Pull request merged**